identity-and-access-management/secret-manager/concepts.mdx

```diff
@@ -8,6 +8,38 @@ content:
 tags: secret-manager secret version
 ---
 
+
+<Concept>
+
+## Folder
+
+Folders are the underlying structure for organizing and grouping your secrets in one place. Folders are always within a [path](#path).
+
+</Concept>
+
+<Concept>
+
+## Path
+
+A path refers to the access path to your folders and secrets. They contain your folders, secrets and versions. Each path is prefixed with a slash.
```

Comment:
```suggestion
A path refers to the access path to your folders and secrets. Paths contain your folders, secrets and versions. Each path is prefixed with a slash.
```

Reply:
(otherwise I think its confusing, with the previous sentence ending with folders & secrets)